features/parameter-types/rooms.js

```diff
@@ -20,7 +33,7 @@ class Room {
 // See: https://github.com/zinc-collective/convene/issues/41
 defineParameterType({
   name: "accessLevel",
-  regexp: /(Unlocked|Internal|Locked)/,
+  regexp: /(an |a )?(Unlocked|Internal|Locked)/,
```

Comment:
We will probably want to update the arguments we’re passing to the transformer here, since it seems like there will be _two_ arguments in cases where an article adjective is provided, and _one_ argument in cases where only `(Unlocked|Internal|Locked)` are provided.

We may want to think a bit more about how to use parameter types so that we don’t run into this rough edge in the future (for instance, we may want to _require_ the article adjectives, so that we don’t have some cases where a single argument is passed into the classes inflated by the transformer.)